stratum/hal/lib/tdi/tdi_table_manager.cc

```diff
@@ -342,7 +342,7 @@ ::util::Status TdiTableManager::BuildTableData(
                  << "Unsupported meter spec on meter "
                  << meter.ShortDebugString() << ".";
       }
-      TdiPktModMeterConfig config = {0};
+      TdiPktModMeterConfig config;
```

Comment:
This change looks wrong to me. What was wrong previously?

Reply:
used memset